pkg/cmd/search/code/code.go

```diff
@@ -186,14 +188,10 @@ func formatMatch(t string, matches []search.Match, colorize bool) []string {
 			continue
 		}
 		if _, ok := startIndices[i]; ok {
-			if colorize {
-				b.WriteString("\x1b[30;43m") // black text on yellow background
-			}
+			b.WriteString(cs.HighlightStart()) // black text on yellow background
```

Comment:
In making this style reusable, I don't think keeping the original comment describing this makes sense especially being changable.

```suggestion
			b.WriteString(cs.HighlightStart())
```